course3/homework-list/build/.gitignore

```diff
@@ -0,0 +1,3 @@
+*
+!.gitignore
```

Comment:
This way you will aways have the 'build' directory in your source, only to hold this .gitignore.

It would be much nicer _not_ to use in-source build, but rather create the build directory outside of the tree entirely.